resources/kyverno/admission-control-policy/admission-control-verify-image.cue

```diff
@@ -27,10 +28,8 @@ clusterPolicy: "verify-image": {
 				KsXF+jAKBggqhkjOPQQDAwNpADBmAjEAj1nHeXZp+13NWBNa+EDsDP8G1WWg1tCM
 				WP/WHPqpaVo0jhsweNFZgSs0eE7wYI4qAjEA2WB9ot98sIkoF3vZYdd3/VtWB5b9
 				TNMea7Ix/stJ5TfcLLeABLE4BNJOsQ4vnBHJ
-				-----END CERTIFICATE-----
+				-----END CERTIFICATE----
```

Comment:
Not sure I understand this change. Seems like five `-` at the end would be consistent with five `-` at the start.